push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/token/TokenLoader.java

```diff
@@ -107,7 +98,6 @@ public void loadAndQueueTokenBatch(@Observes @Dequeue MessageHolderWithVariants
         final Collection<Variant> variants = msg.getVariants();
```

Comment:
Update comment of the method. This statement "Additionally it fires {@link BatchLoadedEvent} as CDI event (that is translated to JMS event)."  is not valid anymore. The same for "When all batches were loaded for the given variant, it fires  {@link AllBatchesLoadedEvent}."